pages/[league]/index.tsx

```diff
@@ -59,7 +59,7 @@ function LeagueHome({ league }: Props): JSX.Element {
       <Header league={league} />
       <Container>
         <YoutubeEmbedContainer>
-          <LiveStream isSHL={league === 'shl'} />
```

Comment:
added support for all leagues, not just SHL and not-SHL.

Currently, there is no change to the end user, as the international leagues use the SMJHL video id.